test/Bugs/testSimdManyVars.js

```diff
@@ -0,0 +1,371 @@
+function asmModule(stdlib, imports) {
+    "use asm";
+    var i4 = stdlib.SIMD.Int32x4;
+    var i4check = i4.check;
+    var i4splat = i4.splat;
+    
+    var i4fromFloat32x4 = i4.fromFloat32x4;
+    var i4fromFloat32x4Bits = i4.fromFloat32x4Bits;
+    //var i4abs = i4.abs;
+    var i4neg = i4.neg;
+    var i4add = i4.add;
+    var i4sub = i4.sub;
+    var i4mul = i4.mul;
+    //var i4swizzle = i4.swizzle;
+    //var i4shuffle = i4.shuffle;
+    var i4lessThan = i4.lessThan;
+    var i4equal = i4.equal;
+    var i4greaterThan = i4.greaterThan;
+    var i4select = i4.select;
+    var i4and = i4.and;
+    var i4or = i4.or;
+    var i4xor = i4.xor;
+    var i4not = i4.not;
+    //var i4shiftLeftByScalar = i4.shiftLeftByScalar;
+    //var i4shiftRightByScalar = i4.shiftRightByScalar;
+    //var i4shiftRightArithmeticByScalar = i4.shiftRightArithmeticByScalar;
+    
+    var f4 = stdlib.SIMD.Float32x4;  
+    var f4check = f4.check;
+    var f4splat = f4.splat;
+    
+    var f4fromInt32x4 = f4.fromInt32x4;
+    var f4fromUint32x4 = f4.fromUint32x4;
+    var f4fromInt32x4Bits = f4.fromInt32x4Bits;
+    
+    var f4fromInt16x8Bits = f4.fromInt16x8Bits;
+    var f4fromInt8x16Bits = f4.fromInt8x16Bits;
+    var f4fromUint32x4Bits = f4.fromUint32x4Bits;
+    var f4fromUint16x8Bits = f4.fromUint16x8Bits;
+    var f4fromUint8x16Bits = f4.fromUint8x16Bits;
+
+    

```

Comment:
@dilijev parachutes :-) lemme add the copyright header and deal with whitespaces